hail/python/hail/experimental/vcf_combiner.py

```diff
@@ -129,10 +128,7 @@ def min_rep(locus, ref, alt):
     combined = combine(ts)
     combined = combined.annotate(alleles=fix_alleles(combined.alleles))
     return hl.MatrixTable(
```

Comment:
If we don't need the `MatrixKeyRowsBy` then we don't need to construct a new `MatrixTable`, `_unlocalize_entries` will take care of it.

Reply:
Ah, yes, good catch, fixed.